rfcs/0130-stalled-rfcs.md

```diff
@@ -0,0 +1,145 @@
+If an RFC is unable to find adaqite number of shepherds in 2 months it will be considered "low interest". The RFC will be closed to indicate that there is no expection of continuous forward progress.
+
+After 1 month of being open the following message will be added to the PR to suggest ways to find more support:
+
+> This RFC has not acquired enough shepherds. This typically shows lack of interest from the community. In order to progress [a full shepherd team is required](https://github.com/NixOS/rfcs/#shepherd-team). Consider trying to raise interest by [posting in Discourse](https://discourse.nixos.org/), [taking in Matrix](https://matrix.to/#/#community:nixos.org) or reaching out to people that you know.
+>
+> If not enough shepherds can be found in the next month we will close this RFC until we can find enough interested participants. The PR can be reopened at any time if more shepherd nominations are made.
+>
+> [See more info on the Nix RFC process here](https://github.com/NixOS/rfcs/blob/master/README.md)
+
+If another month has elapsed since the warning without sufficent accepted nominations the following message will be posted, the `status: insufficent interest` label will be added and the issue will be closed.
+
+> This RFC is being closed due to lack interest. If enough shepherds are found this issue can be reopened. If you don't have permission to reopen please [open an issue for the NixOS RFC Steering Committee](https://github.com/NixOS/rfc-steering-committee/issues/new) linking to this PR.
+
+## Renewed Interest
+
+If an RFC that was closed due to lack of interest finds sufficent interest it can be reopened and the `status: insufficent interest` label removed. The RFC will then continue through the RFC process as normal.
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+## More Research Required
+
+If an RFC is opened and discussion reveals that more research is required the author may now return their RFC to draft status to indicate that they are working on an update.
+
+This is already done frequently done today but this RFC codifies this process. The alternative which is sometimes seen is that the NixOS RFC Steering Committee keeps pinging the RFC for months trying to move the RFC forward which is annoying and unproductive.
+
+## Can't Find Shepherds
+
+If an RFC is opened but a team of shepherds can't be assembed the RFC will now be closed.
+
+Currently there is no process here and the NixOS RFC Steering Committee is unsure what to do and tends to annoyingly ping these RFCs with no positive outcome for a few months then improvise. This RFC proposes a well-defined process to ensure that everyone understands the process.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+## Closing a PR may be demotivating.
+
+Strugging to find shepherds for an RFC may already be demotivating and having the RFC closed can be an aditional flag that the RFC is strugging. This may be demoralizing to the RFC author.
```

Comment:
```suggestion
Struggling to find shepherds for an RFC may already be demotivating and having the RFC closed can be an additional flag that the RFC is struggling. This may be demoralizing to the RFC author.
```